spyder/plugins/editor/widgets/editor.py

```diff
@@ -287,10 +286,17 @@ def __init__(self, parent, actions):
         self.setLayout(layout)
 
         self.menu = None
-        self.switcher_dlg = None
         self.switcher_manager = None
         self.tabs = None
         self.tabs_switcher = None
+        self.switcher_plugin = None
```

Comment:
I think for simplicity we could rename this attribute to be `switcher`. What do you say @dalthviz?

Reply:
I think leaving `switcher_plugin` is okay since that could be handy in the future (to know that the reference here is to the Switcher plugin by just checking the variable name)